recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "coiled-runtime" %}
-{% set version = "0.1.0" + environ.get("VERSION_SUFFIX", '') %}
-{% set dask_version = environ.get("DASK_VERSION", "2022.6.0") %}
-{% set distributed_version = environ.get("DISTRIBUTED_VERSION", "2022.6.0") %}
+{% set version = "0.2.0" + environ.get("VERSION_SUFFIX", '') %}
```

Comment:
What's this version number?

Reply:
This goes into the release number for nightly builds for `coiled-runtime` on the `coiled` conda channel. Thinking about this more, this should probably be updated to be `0.0.1` instead of `0.2.0`